torchvision/transforms/functional.py

```diff
@@ -223,9 +223,11 @@ def to_pil_image(pic, mode=None):
             pic = np.expand_dims(pic, 2)
 
     npimg = pic
-    if isinstance(pic, torch.FloatTensor) and mode != 'F':
+    if pic.is_floating_point() and mode != 'F':
```

Comment:
This breaks support for `np.ndarray` as it doesn't have support for `is_floating_point`